refs/schemas/priv/RFC-PRIV.md

```diff
@@ -0,0 +1,893 @@
+| `data-reference` | 1-* | one or more references that uniquely identify the data that the capture concerns (e.g. a legal case file reference, account ID, contract ID, a URL)|
+| `target` | 0-1 | [Target Terms](#target-terms). In absence of indication `SYSTEM` is assumed |
+| `fragments` | 1-* | One or more [Data Capture Fragments](#data-capture-fragments) |
+
+A Data Capture concerns one and only one Data Subject who CAN be identified by multiple Data Subject Identities.
+
+#### Data Capture Fragments
+
+| Property | Expected cardinality | Expected values |
+| --------------- | ------ | -------------------- |
+| `fragment-id` | 1 | a string in the [uuid](https://www.rfc-editor.org/rfc/rfc4122.html) format |
+| `selector` | 1 | a string used to uniquely identify a data field (in the System's data model) to which the fragment corresponds. MUST be a subcategory of a [Data Category](#data-categories) and MUST be defined according to [Term Dot Notation](#term-dot-notation) |
+| `date` | 1 | Date and Time when data was Captured was given in JSON Schema [date-time](https://json-schema.org/draft/2020-12/json-schema-validation.html#rfc.section.7.3.1) format |
+| `scope` |  0-1 | a [Privacy Scope](#privacy-scope) in absence of which the fragment SHOULD be interpreted as unlimited, including all categories of all dimensions |
+| `target` | 0-1 | [Target Terms](#target-terms). In absence of indication `SYSTEM` is assumed |
+| `retention` | 1-* | one or more [Retention Policies](#retention-policy) |
+| `provenance` | 1-* | one or more [Provenance](#provenance) to indicate how the data was obtained |
+| `data` | 0-* | Optionally concrete data |
+| `legal-base` | 0-* | [Legal Bases](#legal-bases) |
+
+A `selector` MUST include, at the beginning of its string, one of the [Data Category Terms](#data-categories).
+Its syntax MUST follow the [Term Dot Notation](#term-dot-notation).
+A `selector` is considered a subcategory (in the sense of the [Term Dot Notation](#term-dot-notation)) of the most granular [Data Category Term](#data-categories) that it includes at at the beginning of its string.
+
+For example selectors 'CONTACT.ADDRESS.SHIPPING' and 'CONTACT.ADDRESS.BILLING' indicate that the data being captured by a particular fragment belonging to the `CONTACT.ADDRESS` [Data Category](#data-categories) of which they are considered to be a subcategory.
+
+While the Data Categories are globally defined by PRIV and interpreted the same way by all implementing Systems, the selectors are defined by the Systems.
+A `selector` uniquely identifies a particular data field that the Systems works with.
+When several Systems exchange data among them, they SHOULD align on using the same `selectors` in the same way, in order to be able to correctly interoperate.
+
+The value of the `target` indicates whether the data in question is going to be processed only by the System collecting it, of within a larger target scope. See [Targets](#targets).
```

Comment:
>  Is this indicated on the time of capture?

Yes. Caputre time. Also at caputre time the System can inform the user about the target for which the data is being captured. If consent is used as legal base, then consent explicitely includes SHARING

>     Can it be changed later?

This can happen. The Organziation can change the Privacy Policy, but then they MUST inform the users about it. So we should support the scneario where the Organziation, changes the Target property of past caputrs simultaneousely with a Privacy Policy change notification.

>     Should we refuse the transfer if target=SYSTEM?

It is not the role of blindnet devkit to enforce compliance. It is the role of blindnet devkit to help those who want to comply, to do so easily. So, NO.